clj-refactor.el

```diff
@@ -1054,14 +1054,18 @@ word test in it and whether the file lives under the test/ directory."
       (s-ends-with? ".cljc" (buffer-file-name))))
 
 (defun cljr--add-ns-if-blank-clj-file ()
+  (interactive)
```

Comment:
Yes, sorry I was clear @aiba. What I meant before was that you don't need this `(interactive)` at all.